tools/StatsAnalyzer/Nethermind.StatsAnalyzer.Plugin/Tracer/Call/CallAnalyzerTraceEntry.cs

```diff
@@ -0,0 +1,14 @@
+using System.Text.Json.Serialization;
+using Nethermind.Serialization.Json;
+
+namespace Nethermind.StatsAnalyzer.Plugin.Tracer.Call;
+
+public class CallAnalyzerTraceEntry
+{
+    [JsonPropertyName("address")]
```

Comment:
I think that JsonPropertyName when it is same name but lowercase are not really needed?